### DISCUSSION TITLE
Unable to connect to Postgres and fetch tables - "this.client.close is not a function"

### DISCUSSION THREAD

**Discussion by mathewhealthi** [Budibase/budibase#5791] [Help]
While trying to connect to our hosted Postgres database, the error '**this.client.close is not a function**' is appearing. We've whitelisted the required IP address to access the database, yet this error persists. Please advice.

**Comment by melohagan** [comment]
Hi @mathewhealthi 

This appears to be a bug; another user has raised the issue here: https://github.com/Budibase/budibase/issues/5792
